src/components/core/DpBulkEditHeader.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpBulkEditHeader from './DpBulkEditHeader.vue'
+
+const meta: Meta<typeof DpBulkEditHeader> = {
+    component: DpBulkEditHeader,
+    title: 'Components/BulkEditHeader',
+    render: (args) => ({
+        components: {
+            DpBulkEditHeader
+        },
+        computed: {
+            selectedItemsCount () {
+                return args.options.filter(checked => checked.checked == true)
+            }
+        },
+        setup() {
+            const toggleSelectAll = () => {
+                for (let element of args.options) {
+                    if(element.checked === false) {
+                        element.checked = true
+                    } else {
+                        element.checked = false
+                    }
+                }
+            }
+
+            const resetSelection = () => {
+                for (const element of args.options) {
+                    element.checked = false
+                }
+            }
+
+            const deleteSelection = () => {
+                for (let element of args.options) {
+                    if(element.checked === true) {
+                        args.options = args.options.filter(checked => checked.checked != true)
+                    }
+                }
+            }
+
+            return { args, deleteSelection, resetSelection, toggleSelectAll }
+        },
+        template: `
+          <div>
+              <dp-bulk-edit-header
+                  v-if="selectedItemsCount.length > 0"
+                  :selected-items-count="selectedItemsCount.length"
+                  selection-text="Elements"
```

Comment:
This is a weakness of the component that i'd like to fix, maybe within this same PR: The text that is actually displayed is [computed within DpBulkEditHeader](https://github.com/demos-europe/demosplan-ui/blob/041435553f49b4692b99696a68e58493066460f3/src/components/core/DpBulkEditHeader.vue#L42), from the two props that are also passed to DpBulkEditHeader. However this way we are not able to generate custom plural forms. I'd suggest to have just one prop `selectedItemsText` (and drop the `selectedItemsCount` in DpBulkEditHeader). Then we would move the count to the translations in demosplan-core, to enable nice plural forms.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/cb41626c68f765620949f45d396957f14379cd3b

Reply:
see also: https://github.com/demos-europe/demosplan-core/pull/2245